utils/util.go

```diff
@@ -39,6 +39,13 @@ const (
 	STRAYSENSOR
 )
 
+type RequestFilters struct {
+	FieldsToShow []string `schema:"fieldOnly"`
+	StartDate    []string `schema:"startDate"`
```

Comment:
Only the first elements are accessed, so I find carrying slices a bit confusing. Maybe we can reduce them when the struct is created in getFiltersFromQueryParams.

Reply:
I thought we may find it useful if later we implement multiple date ranges (from March 2 until 5 or March 15 until 17, for example, something the date component in flutter can easily do), but since it's not like that now I'll change it. A simple change in the struct should do it. 
For the ones that no longer are slices, if more than one is sent in the query (?limit=2&limit=3), only the last one (limit=3) will be used from now.